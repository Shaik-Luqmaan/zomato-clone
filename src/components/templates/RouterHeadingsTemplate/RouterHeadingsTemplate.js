import React from "react";
import { makeStyles } from "@material-ui/core/styles";
import { Grid } from "@material-ui/core";
import Typography from "../../atoms/Typography/Typography";
import Collections from "../CollectionsTemplate/Collections";
import { Link } from "react-router-dom";

const useStyles = makeStyles({
    root: {
      paddingLeft: "10px"
    },
    subHeading: {
      color: "rgb(100,100,100,.9)",
      cursor: "pointer",
      padding:"10px 10px",
    },
    active: {
      color: "rgb(240,0,0,.7)",
      borderBottom: "2px solid rgb(240,0,0,.4)",
      padding: "20px 10px 40px 30px",
    },
    subHeadingBar: {
      padding: "0 0 60px 60px",
    },
    link: {
        textDecoration: "none",
        color: "inherit",
      },
    text:{
       padding: "0 30px",
    },
    background : {
        background: "rgb(248, 248, 248)",
        padding: "10px 10px 10px 10px",
    },
    tab:{
        border: "1px solid rgb(207, 207, 207)",
        color: "rgb(156, 156, 156)",
        backgroundColor: "rgb(255, 255, 255)",
        boxShadow: "rgba(54, 54, 54, 0.06) 0px 5px 3px",
        borderRadius: "0.4rem",
        display: "flex",
        alignItems: "center",
        justifyContent: "center",
        cursor: "pointer",
        height: "35px",
        width:"90px",
        margin: "20px 0 20px 30px",
        fontSize: "30px"
    }
  });

function Headings(props) {
    const classes = useStyles();

    let activeType = 0;
    if (props.heading) {
      activeType = props.heading;
    }

    return(
        <Grid container direction="column" className={classes.root}>
            <Grid item  direction="row" container spacing={8} className={classes.subHeadingBar}>
                {headingsList.map((headingItem, index) => {
                let classList = classes.subHeading;
                if (index === activeType) {
                classList += " " + classes.active;
                }
                return (
                    <Grid item className={classList} key={index}>
                         <Link
                            to={"/hyderabad/" + headingItem.link}
                            className={classes.link}>
                            <Typography variant="h6">{headingItem.title}</Typography>
                        </Link>
                    </Grid>
                );
            })}
            </Grid>
        
            {activeType > 0 && (<Grid items><Collections /></Grid>)}

        <Grid container direction="row" >    
            {tabList.map((tab,index) =>{
                return(
                    <Grid items className={classes.tab}>
                        <Typography variant="subtitle1">{tab.name}</Typography>
                    </Grid> 
                )
            })}           
        </Grid>
            <Grid item className={classes.text}>
                <Typography variant="h4">
                    {headingsList[activeType].title} Restaurants in Hyderabad
                </Typography>
            </Grid>
        </Grid>    
    )
}

const headingsList = [
    {
      title: "Delivery",
      link: "delivery"
    },
    {
      title: "Dining Out",
      link:"dine-out"
    },
    {
      title: "Nightlife",
      link: "night-life"
    },
];

const tabList = [
    {
        name: "Filter"
    },
    {
        name: "Offers"
    },
    {
        name: "Distance"
    }, 
    {
        name: "Online"
    },

]

export default Headings



 