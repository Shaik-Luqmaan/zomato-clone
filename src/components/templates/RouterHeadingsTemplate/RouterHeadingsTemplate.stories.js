import React from "react";
import HeadingsTemplate from "./RouterHeadingsTemplate";
import { BrowserRouter } from 'react-router-dom';

export default {
    title: "templates/Headings",
    component: HeadingsTemplate,
}


const Template = ({...args})=>(
    <BrowserRouter> <HeadingsTemplate {...args} /> </BrowserRouter>
);


export const Delivery = Template.bind({});
Delivery.args = {
    heading: 0,
}

export const Dineout = Template.bind({});
Dineout.args = {
    heading: 1,
}

export const Nightlife = Template.bind({});
Nightlife.args = {
    heading: 2,
}