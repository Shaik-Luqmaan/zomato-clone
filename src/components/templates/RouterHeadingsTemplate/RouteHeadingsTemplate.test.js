import React from "react";
import ReactDOM from "react-dom";
import { BrowserRouter } from "react-router-dom";
import "@testing-library/jest-dom";
import RouterHeadingsTemplate from "./RouterHeadingsTemplate";
import renderer from "react-test-renderer";


it("renders RouterHeadingsTemplate without crashing ", () => {
    const div = document.createElement("div");
    ReactDOM.render(
      <BrowserRouter>
        <RouterHeadingsTemplate />
      </BrowserRouter>,
      div
    );
  });
  

it("renders snapshot correctly 1", () => {
  const tree = renderer
    .create(
      <BrowserRouter>
        <RouterHeadingsTemplate />
      </BrowserRouter>
    )
    .toJSON();
  expect(tree).toMatchSnapshot();
});

it("render snapshot correctly 2", () => {
  const tree = renderer
    .create(
      <BrowserRouter>
        <RouterHeadingsTemplate heading="1" />
      </BrowserRouter>
    )
    .toJSON();
  expect(tree).toMatchSnapshot();
});

it("renders snapshot correctly 3", () => {
  const tree = renderer
    .create(
      <BrowserRouter>
        <RouterHeadingsTemplate heading="2" />
      </BrowserRouter>
    )
    .toJSON();
  expect(tree).toMatchSnapshot();
});