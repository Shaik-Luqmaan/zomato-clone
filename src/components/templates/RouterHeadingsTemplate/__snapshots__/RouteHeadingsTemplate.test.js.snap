// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`render snapshot correctly 2 1`] = `
<div
  className="MuiGrid-root makeStyles-root-1 MuiGrid-container MuiGrid-direction-xs-column"
>
  <div
    className="MuiGrid-root makeStyles-subHeadingBar-4 MuiGrid-container MuiGrid-spacing-xs-8 MuiGrid-item"
  >
    <div
      className="MuiGrid-root makeStyles-subHeading-2 MuiGrid-item"
    >
      <a
        className="makeStyles-link-5"
        href="/hyderabad/delivery"
        onClick={[Function]}
      >
        <h6
          className="MuiTypography-root MuiTypography-h6"
        >
          Delivery
        </h6>
      </a>
    </div>
    <div
      className="MuiGrid-root makeStyles-subHeading-2 MuiGrid-item"
    >
      <a
        className="makeStyles-link-5"
        href="/hyderabad/dine-out"
        onClick={[Function]}
      >
        <h6
          className="MuiTypography-root MuiTypography-h6"
        >
          Dining Out
        </h6>
      </a>
    </div>
    <div
      className="MuiGrid-root makeStyles-subHeading-2 MuiGrid-item"
    >
      <a
        className="makeStyles-link-5"
        href="/hyderabad/night-life"
        onClick={[Function]}
      >
        <h6
          className="MuiTypography-root MuiTypography-h6"
        >
          Nightlife
        </h6>
      </a>
    </div>
  </div>
  <div
    className="MuiGrid-root"
    items={true}
  >
    <div
      className="MuiGrid-root makeStyles-root-9 MuiGrid-container MuiGrid-direction-xs-column"
    >
      <div
        className="MuiGrid-root makeStyles-title-12 MuiGrid-item"
      >
        <h4
          className="MuiTypography-root MuiTypography-h4"
        >
          Collections
        </h4>
      </div>
      <div
        className="MuiGrid-root MuiGrid-container MuiGrid-item"
      >
        <div
          className="MuiGrid-root makeStyles-text-11 MuiGrid-item MuiGrid-grid-lg-true"
        >
          <h6
            className="MuiTypography-root MuiTypography-h6"
          >
            Explore curated lists of top restaurants, cafes, pubs, and bars in Hyderabad, based on trends.
          </h6>
        </div>
        <div
          className="MuiGrid-root makeStyles-link-10 MuiGrid-container MuiGrid-item MuiGrid-align-items-xs-flex-end MuiGrid-justify-xs-flex-end MuiGrid-grid-lg-3"
        >
          <div
            className="MuiGrid-root MuiGrid-item"
          >
            <h6
              className="MuiTypography-root MuiTypography-subtitle1"
              mode="link"
            >
              All Collections in Hyderabad
            </h6>
          </div>
          <div
            className="MuiGrid-root MuiGrid-item"
          >
            <svg
              aria-hidden={true}
              className="MuiSvgIcon-root MuiSvgIcon-fontSizeSmall"
              focusable="false"
              viewBox="0 0 24 24"
            >
              <path
                d="M10 17l5-5-5-5v10z"
              />
            </svg>
          </div>
        </div>
      </div>
      <div
        className="MuiGrid-root makeStyles-collection-13 MuiGrid-item"
      >
        <div
          className="MuiGrid-root makeStyles-root-14 MuiGrid-container MuiGrid-spacing-xs-4"
        >
          <div
            className="MuiGrid-root MuiGrid-item MuiGrid-grid-xs-12 MuiGrid-grid-sm-6 MuiGrid-grid-md-3"
          >
            <div
              className="MuiGrid-root makeStyles-root-15 MuiGrid-container MuiGrid-align-items-xs-flex-end"
              data-testid="main"
              style={
                Object {
                  "backgroundImage": "linear-gradient(rgba(0,0,0, 0.1), rgb(0, 0, 0, .6)), url(sweet.jpg)",
                }
              }
            >
              <div
                className="MuiGrid-root makeStyles-content-18 MuiGrid-item"
              >
                <h6
                  className="MuiTypography-root MuiTypography-h6"
                  data-testid="text"
                >
                  Sweet Tooth 2020
                </h6>
                <h6
                  className="MuiTypography-root MuiTypography-subtitle2"
                  data-testid="place"
                >
                  9 Places
                  <svg
                    aria-hidden={true}
                    className="MuiSvgIcon-root makeStyles-arrow-16 MuiSvgIcon-fontSizeSmall"
                    focusable="false"
                    viewBox="0 0 24 24"
                  >
                    <path
                      d="M10 17l5-5-5-5v10z"
                    />
                  </svg>
                </h6>
              </div>
            </div>
          </div>
          <div
            className="MuiGrid-root MuiGrid-item MuiGrid-grid-xs-12 MuiGrid-grid-sm-6 MuiGrid-grid-md-3"
          >
            <div
              className="MuiGrid-root makeStyles-root-15 MuiGrid-container MuiGrid-align-items-xs-flex-end"
              data-testid="main"
              style={
                Object {
                  "backgroundImage": "linear-gradient(rgba(0,0,0, 0.1), rgb(0, 0, 0, .6)), url(most.jpg)",
                }
              }
            >
              <div
                className="MuiGrid-root makeStyles-content-18 MuiGrid-item"
              >
                <h6
                  className="MuiTypography-root MuiTypography-h6"
                  data-testid="text"
                >
                  Most Searched 2020
                </h6>
                <h6
                  className="MuiTypography-root MuiTypography-subtitle2"
                  data-testid="place"
                >
                  28 Places
                  <svg
                    aria-hidden={true}
                    className="MuiSvgIcon-root makeStyles-arrow-16 MuiSvgIcon-fontSizeSmall"
                    focusable="false"
                    viewBox="0 0 24 24"
                  >
                    <path
                      d="M10 17l5-5-5-5v10z"
                    />
                  </svg>
                </h6>
              </div>
            </div>
          </div>
          <div
            className="MuiGrid-root MuiGrid-item MuiGrid-grid-xs-12 MuiGrid-grid-sm-6 MuiGrid-grid-md-3"
          >
            <div
              className="MuiGrid-root makeStyles-root-15 MuiGrid-container MuiGrid-align-items-xs-flex-end"
              data-testid="main"
              style={
                Object {
                  "backgroundImage": "linear-gradient(rgba(0,0,0, 0.1), rgb(0, 0, 0, .6)), url(trending.jpg)",
                }
              }
            >
              <div
                className="MuiGrid-root makeStyles-content-18 MuiGrid-item"
              >
                <h6
                  className="MuiTypography-root MuiTypography-h6"
                  data-testid="text"
                >
                  Best Openings of 2020
                </h6>
                <h6
                  className="MuiTypography-root MuiTypography-subtitle2"
                  data-testid="place"
                >
                  25 Places
                  <svg
                    aria-hidden={true}
                    className="MuiSvgIcon-root makeStyles-arrow-16 MuiSvgIcon-fontSizeSmall"
                    focusable="false"
                    viewBox="0 0 24 24"
                  >
                    <path
                      d="M10 17l5-5-5-5v10z"
                    />
                  </svg>
                </h6>
              </div>
            </div>
          </div>
          <div
            className="MuiGrid-root MuiGrid-item MuiGrid-grid-xs-12 MuiGrid-grid-sm-6 MuiGrid-grid-md-3"
          >
            <div
              className="MuiGrid-root makeStyles-root-15 MuiGrid-container MuiGrid-align-items-xs-flex-end"
              data-testid="main"
              style={
                Object {
                  "backgroundImage": "linear-gradient(rgba(0,0,0, 0.1), rgb(0, 0, 0, .6)), url(collection_1.jpg)",
                }
              }
            >
              <div
                className="MuiGrid-root makeStyles-content-18 MuiGrid-item"
              >
                <h6
                  className="MuiTypography-root MuiTypography-h6"
                  data-testid="text"
                >
                  Trending this Week
                </h6>
                <h6
                  className="MuiTypography-root MuiTypography-subtitle2"
                  data-testid="place"
                >
                  28 Places
                  <svg
                    aria-hidden={true}
                    className="MuiSvgIcon-root makeStyles-arrow-16 MuiSvgIcon-fontSizeSmall"
                    focusable="false"
                    viewBox="0 0 24 24"
                  >
                    <path
                      d="M10 17l5-5-5-5v10z"
                    />
                  </svg>
                </h6>
              </div>
            </div>
          </div>
        </div>
      </div>
    </div>
  </div>
  <div
    className="MuiGrid-root makeStyles-text-6 MuiGrid-item"
  >
    <h4
      className="MuiTypography-root MuiTypography-h4"
    >
      Dining Out
       Restaurants in Hyderabad
    </h4>
  </div>
</div>
`;

exports[`renders snapshot correctly 1 1`] = `
<div
  className="MuiGrid-root makeStyles-root-1 MuiGrid-container MuiGrid-direction-xs-column"
>
  <div
    className="MuiGrid-root makeStyles-subHeadingBar-4 MuiGrid-container MuiGrid-spacing-xs-8 MuiGrid-item"
  >
    <div
      className="MuiGrid-root makeStyles-subHeading-2 makeStyles-active-3 MuiGrid-item"
    >
      <a
        className="makeStyles-link-5"
        href="/hyderabad/delivery"
        onClick={[Function]}
      >
        <h6
          className="MuiTypography-root MuiTypography-h6"
        >
          Delivery
        </h6>
      </a>
    </div>
    <div
      className="MuiGrid-root makeStyles-subHeading-2 MuiGrid-item"
    >
      <a
        className="makeStyles-link-5"
        href="/hyderabad/dine-out"
        onClick={[Function]}
      >
        <h6
          className="MuiTypography-root MuiTypography-h6"
        >
          Dining Out
        </h6>
      </a>
    </div>
    <div
      className="MuiGrid-root makeStyles-subHeading-2 MuiGrid-item"
    >
      <a
        className="makeStyles-link-5"
        href="/hyderabad/night-life"
        onClick={[Function]}
      >
        <h6
          className="MuiTypography-root MuiTypography-h6"
        >
          Nightlife
        </h6>
      </a>
    </div>
  </div>
  <div
    className="MuiGrid-root makeStyles-text-6 MuiGrid-item"
  >
    <h4
      className="MuiTypography-root MuiTypography-h4"
    >
      Delivery
       Restaurants in Hyderabad
    </h4>
  </div>
</div>
`;

exports[`renders snapshot correctly 3 1`] = `
<div
  className="MuiGrid-root makeStyles-root-1 MuiGrid-container MuiGrid-direction-xs-column"
>
  <div
    className="MuiGrid-root makeStyles-subHeadingBar-4 MuiGrid-container MuiGrid-spacing-xs-8 MuiGrid-item"
  >
    <div
      className="MuiGrid-root makeStyles-subHeading-2 MuiGrid-item"
    >
      <a
        className="makeStyles-link-5"
        href="/hyderabad/delivery"
        onClick={[Function]}
      >
        <h6
          className="MuiTypography-root MuiTypography-h6"
        >
          Delivery
        </h6>
      </a>
    </div>
    <div
      className="MuiGrid-root makeStyles-subHeading-2 MuiGrid-item"
    >
      <a
        className="makeStyles-link-5"
        href="/hyderabad/dine-out"
        onClick={[Function]}
      >
        <h6
          className="MuiTypography-root MuiTypography-h6"
        >
          Dining Out
        </h6>
      </a>
    </div>
    <div
      className="MuiGrid-root makeStyles-subHeading-2 MuiGrid-item"
    >
      <a
        className="makeStyles-link-5"
        href="/hyderabad/night-life"
        onClick={[Function]}
      >
        <h6
          className="MuiTypography-root MuiTypography-h6"
        >
          Nightlife
        </h6>
      </a>
    </div>
  </div>
  <div
    className="MuiGrid-root"
    items={true}
  >
    <div
      className="MuiGrid-root makeStyles-root-9 MuiGrid-container MuiGrid-direction-xs-column"
    >
      <div
        className="MuiGrid-root makeStyles-title-12 MuiGrid-item"
      >
        <h4
          className="MuiTypography-root MuiTypography-h4"
        >
          Collections
        </h4>
      </div>
      <div
        className="MuiGrid-root MuiGrid-container MuiGrid-item"
      >
        <div
          className="MuiGrid-root makeStyles-text-11 MuiGrid-item MuiGrid-grid-lg-true"
        >
          <h6
            className="MuiTypography-root MuiTypography-h6"
          >
            Explore curated lists of top restaurants, cafes, pubs, and bars in Hyderabad, based on trends.
          </h6>
        </div>
        <div
          className="MuiGrid-root makeStyles-link-10 MuiGrid-container MuiGrid-item MuiGrid-align-items-xs-flex-end MuiGrid-justify-xs-flex-end MuiGrid-grid-lg-3"
        >
          <div
            className="MuiGrid-root MuiGrid-item"
          >
            <h6
              className="MuiTypography-root MuiTypography-subtitle1"
              mode="link"
            >
              All Collections in Hyderabad
            </h6>
          </div>
          <div
            className="MuiGrid-root MuiGrid-item"
          >
            <svg
              aria-hidden={true}
              className="MuiSvgIcon-root MuiSvgIcon-fontSizeSmall"
              focusable="false"
              viewBox="0 0 24 24"
            >
              <path
                d="M10 17l5-5-5-5v10z"
              />
            </svg>
          </div>
        </div>
      </div>
      <div
        className="MuiGrid-root makeStyles-collection-13 MuiGrid-item"
      >
        <div
          className="MuiGrid-root makeStyles-root-14 MuiGrid-container MuiGrid-spacing-xs-4"
        >
          <div
            className="MuiGrid-root MuiGrid-item MuiGrid-grid-xs-12 MuiGrid-grid-sm-6 MuiGrid-grid-md-3"
          >
            <div
              className="MuiGrid-root makeStyles-root-15 MuiGrid-container MuiGrid-align-items-xs-flex-end"
              data-testid="main"
              style={
                Object {
                  "backgroundImage": "linear-gradient(rgba(0,0,0, 0.1), rgb(0, 0, 0, .6)), url(sweet.jpg)",
                }
              }
            >
              <div
                className="MuiGrid-root makeStyles-content-18 MuiGrid-item"
              >
                <h6
                  className="MuiTypography-root MuiTypography-h6"
                  data-testid="text"
                >
                  Sweet Tooth 2020
                </h6>
                <h6
                  className="MuiTypography-root MuiTypography-subtitle2"
                  data-testid="place"
                >
                  9 Places
                  <svg
                    aria-hidden={true}
                    className="MuiSvgIcon-root makeStyles-arrow-16 MuiSvgIcon-fontSizeSmall"
                    focusable="false"
                    viewBox="0 0 24 24"
                  >
                    <path
                      d="M10 17l5-5-5-5v10z"
                    />
                  </svg>
                </h6>
              </div>
            </div>
          </div>
          <div
            className="MuiGrid-root MuiGrid-item MuiGrid-grid-xs-12 MuiGrid-grid-sm-6 MuiGrid-grid-md-3"
          >
            <div
              className="MuiGrid-root makeStyles-root-15 MuiGrid-container MuiGrid-align-items-xs-flex-end"
              data-testid="main"
              style={
                Object {
                  "backgroundImage": "linear-gradient(rgba(0,0,0, 0.1), rgb(0, 0, 0, .6)), url(most.jpg)",
                }
              }
            >
              <div
                className="MuiGrid-root makeStyles-content-18 MuiGrid-item"
              >
                <h6
                  className="MuiTypography-root MuiTypography-h6"
                  data-testid="text"
                >
                  Most Searched 2020
                </h6>
                <h6
                  className="MuiTypography-root MuiTypography-subtitle2"
                  data-testid="place"
                >
                  28 Places
                  <svg
                    aria-hidden={true}
                    className="MuiSvgIcon-root makeStyles-arrow-16 MuiSvgIcon-fontSizeSmall"
                    focusable="false"
                    viewBox="0 0 24 24"
                  >
                    <path
                      d="M10 17l5-5-5-5v10z"
                    />
                  </svg>
                </h6>
              </div>
            </div>
          </div>
          <div
            className="MuiGrid-root MuiGrid-item MuiGrid-grid-xs-12 MuiGrid-grid-sm-6 MuiGrid-grid-md-3"
          >
            <div
              className="MuiGrid-root makeStyles-root-15 MuiGrid-container MuiGrid-align-items-xs-flex-end"
              data-testid="main"
              style={
                Object {
                  "backgroundImage": "linear-gradient(rgba(0,0,0, 0.1), rgb(0, 0, 0, .6)), url(trending.jpg)",
                }
              }
            >
              <div
                className="MuiGrid-root makeStyles-content-18 MuiGrid-item"
              >
                <h6
                  className="MuiTypography-root MuiTypography-h6"
                  data-testid="text"
                >
                  Best Openings of 2020
                </h6>
                <h6
                  className="MuiTypography-root MuiTypography-subtitle2"
                  data-testid="place"
                >
                  25 Places
                  <svg
                    aria-hidden={true}
                    className="MuiSvgIcon-root makeStyles-arrow-16 MuiSvgIcon-fontSizeSmall"
                    focusable="false"
                    viewBox="0 0 24 24"
                  >
                    <path
                      d="M10 17l5-5-5-5v10z"
                    />
                  </svg>
                </h6>
              </div>
            </div>
          </div>
          <div
            className="MuiGrid-root MuiGrid-item MuiGrid-grid-xs-12 MuiGrid-grid-sm-6 MuiGrid-grid-md-3"
          >
            <div
              className="MuiGrid-root makeStyles-root-15 MuiGrid-container MuiGrid-align-items-xs-flex-end"
              data-testid="main"
              style={
                Object {
                  "backgroundImage": "linear-gradient(rgba(0,0,0, 0.1), rgb(0, 0, 0, .6)), url(collection_1.jpg)",
                }
              }
            >
              <div
                className="MuiGrid-root makeStyles-content-18 MuiGrid-item"
              >
                <h6
                  className="MuiTypography-root MuiTypography-h6"
                  data-testid="text"
                >
                  Trending this Week
                </h6>
                <h6
                  className="MuiTypography-root MuiTypography-subtitle2"
                  data-testid="place"
                >
                  28 Places
                  <svg
                    aria-hidden={true}
                    className="MuiSvgIcon-root makeStyles-arrow-16 MuiSvgIcon-fontSizeSmall"
                    focusable="false"
                    viewBox="0 0 24 24"
                  >
                    <path
                      d="M10 17l5-5-5-5v10z"
                    />
                  </svg>
                </h6>
              </div>
            </div>
          </div>
        </div>
      </div>
    </div>
  </div>
  <div
    className="MuiGrid-root makeStyles-text-6 MuiGrid-item"
  >
    <h4
      className="MuiTypography-root MuiTypography-h4"
    >
      Nightlife
       Restaurants in Hyderabad
    </h4>
  </div>
</div>
`;
